DeepSpeech.py

```diff
@@ -358,8 +358,8 @@ def BiRNN(batch_x, seq_length, dropout):
     # clipped RELU activation and dropout.
 
     # 1st layer
-    b1 = variable_on_worker_level('b1', [n_hidden_1], tf.random_normal_initializer(stddev=FLAGS.b1_stddev))
-    h1 = variable_on_worker_level('h1', [n_input + 2*n_input*n_context, n_hidden_1], tf.random_normal_initializer(stddev=FLAGS.h1_stddev))
+    b1 = variable_on_worker_level('b1', [n_hidden_1], tf.contrib.layers.xavier_initializer(uniform=False))
```

Comment:
After some digging, it seems its not a good idea to initialize bias with xavier_initializer Referenced: https://github.com/tensorflow/tensorflow/issues/3825

Biases can also be initialized as zeros. So, it made sense to me using the same for bias as well. Will revert the change for bias